packages/datadog-plugin-prisma/src/client.js

```diff
@@ -0,0 +1,38 @@
+'use strict'
+const DatabasePlugin = require('../../dd-trace/src/plugins/database')
+
+class PrismaCLientPlugin extends DatabasePlugin {
+  static get id () { return 'prisma' }
+  static get operation () { return 'client' }
+  static get system () { return 'prisma' }
+
+  start (ctx) {
+    const service = this.serviceName({ pluginConfig: this.config })
+    const resource = formatResourceName(ctx.resourceName, ctx.attributes)
+    const options = { service, resource }
+
+    if (ctx.resourceName === 'operation') {
+      options.meta = {
+        prisma: {
+          method: ctx.attributes.method,
+          model: ctx.attributes.model,
+          type: 'client'
+        }
+      }
+    }
+    this.startSpan(this.operationName({ operation: this.operation }), options)
+  }
+}
+
+function formatResourceName (resource, attributes) {
+  if (attributes && attributes.name) {
+    return `${attributes.name || ''}`.trim()
+  }
+  if (attributes && attributes.model && attributes.method) {
+    return `${attributes.model || ''}.${attributes.method || ''}`.trim()
```

Comment:
```suggestion
  if (attributes?.name) {
    return `${attributes.name}`.trim()
  }
  if (attributes?.model && attributes.method) {
    return `${attributes.model}.${attributes.method}`.trim()
```

These are impossible to be empty strings due to the truthy checks in the if statements.

Reply:
Fixed 👍